syntax = "proto3";
package setup;

service Setup {
    rpc AddAccount (AddAccountRequest) returns (AddAccountResponse);
}

message Transaction {
    string trx_id = 1;
}

message Account {
    string branch = 1;
    string accountnumber = 2;
    string accounttype = 3;
    string openingdate = 4;
}

message AddAccountRequest {
    Account account = 1;
}

message AddAccountResponse {
    Transaction transaction = 1;
}
